ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js

```diff
@@ -79,7 +79,7 @@ const ConfirmAddCustodianToken = () => {
                 setShowMore(true);
               }}
             >
-              {t('ShowMore')}
+              {t('showMore')}
```

Comment:
do we have corresponding text to this key?

Reply:
Yes, there was a typo problem